crates/fluvio-protocol/Cargo.toml

```diff
@@ -72,3 +72,4 @@ fluvio-future = { workspace = true, features = [
     "net",
 ] }
 futures = { workspace = true }
+tracing-subscriber = { workspace = true,  features = ["env-filter", "fmt"] }
```

Comment:
why this is needed here?

Reply:
It was a needed dev-dependency to debug w/ some of the tracing messages running unit test during development. But not regularly needed. remove?

Reply:
yes if not needed